frontend/src/components/GroupModal.jsx

```diff
@@ -1,15 +1,39 @@
 import "../styles/GroupModal.css";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import MemberSearch from "./MemberSearch";
+import Select from "react-select";
+import makeAnimated from "react-select/animated";
 
 const defaultGroup = {
   name: "",
   img: "/deafult_group_pic.png",
+  tags: [],
   members: [],
 };
 
+const options = [
```

Comment:
nit: Make the constants be LOUD
```suggestion
const GROUP_TAG_OPTIONS = [
```